autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -392,7 +400,7 @@ def numpy_array_to_pandas(
         Returns:
             pd.DataFrame
         """
-        return pd.DataFrame(X).infer_objects().convert_dtypes()
+        return pd.DataFrame(X).convert_dtypes()
```

Comment:
Could you explain why we can remove `infer_objects`?

Reply:
thats a good point. I think @ArlindKadra made this change, maybe he could clarify as I dont know

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.convert_dtypes.html

Because `convert_dtypes` infers objects by default.